Cargo.toml

```diff
@@ -58,6 +58,10 @@ hickory-resolver = "0.24.0"
 simple_logger = "4.2.0"
 shared_singleton = "0.1.0"
 testing_logger = "0.1.1"
+cookie = { version = "0.18.0", features = ["secure", "private"] }
+reqwest = { version = "0.11.22", features = ["blocking"] }
```

Comment:
we do blocking reqwest for now.. this will be changed into something better later on

Reply:
We had `reqwest` in one of the binaries at one point, and that brought in Tokio.  It was replaced with `ureq`, which is simple and synchronous and doesn't bring in an async runtime.  Is there a reason `ureq` won't be adequate here? I'd pitch for using `ureq` where you're thinking of using `reqwest`.

Reply:
Ureq is prob fine too. Ill change that

Reply:
It looks like it won't be hard.  See the code snippet in a comment below.